linkcheck/checker/mailtourl.py

```diff
@@ -114,11 +119,13 @@ def parse_addresses(self):
         if i < (len(url) - 1):
             self.addresses.update(getaddresses(url[:i]))
             try:
-                headers = urllib.parse.parse_qs(url[(i+1):], strict_parsing=True)
+                headers = urllib.parse.parse_qs(url[(i + 1):], strict_parsing=True)
```

Comment:
Huh.

So, black changes `[:i+1]` to `[: i + 1]`, with the weird (to me) space after `:`, but if the arithmetic expression is parenthesized unnecessarily, Black doesn't remove the parentheses, and it doesn't add the space.